src/profile/components/profileInformation/profileInformationAccountManagementLinkUtils.ts

```diff
@@ -5,8 +5,12 @@ import {
 } from '../../../auth/useProfile';
 import config from '../../../config';
 
+function extractSingleAmr(profile: Profile | null): string {
+  return profile ? profile.amr[0] : '';
```

Comment:
What if the `amr` array is empty? Should a check be made before executing `profile.amr[0]`?

Reply:
This depends on the conversation above.

Reply:
This is also relevant if the `amr` claim can be missing or an empty string (in the case of Tunnistamo, which doesn't provide the amrs in an array but as a string).

Reply:
The string value is converted to an array of single string. But anyway the return value of that function requires "| undefined"